internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -108,6 +108,15 @@ func resourceKustoCluster() *pluginsdk.Resource {
 					},
 				},
 
+				"restrict_outbound_network_access": {
```

Comment:
Rename it to https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/mssql/mssql_server_resource.go#L168